stack.yaml

```diff
@@ -9,3 +9,6 @@ packages:
   - ./persistent-postgresql
   - ./persistent-redis
   - ./persistent-qq
+
+extra-deps:
+  - resourcet-pool-0.1.0.0
```

Comment:
We need this in `stackage` if we're going to depend on it and continuing offering `persistent` as a Stackage library.

Reply:
I just pushed it to Hackage. I'll put in on Stackage later today. I won't merge this PR until that's done

Reply:
@parsonsmatt Ok I'm adding it here: https://github.com/commercialhaskell/stackage/pull/5758

When it gets merged and the library is added to nightly, what should I change in this PR? If I just remove `extra-deps` from `stack.yaml` here, `stack build` will no longer work because `stack.yaml` uses `lts-14.27`, right?

Reply:
We'll need to keep it in `extra-deps`, but any `nightly` checks shouldn't need it starting soon.